lib/travis_dedup.rb

```diff
@@ -20,6 +23,8 @@ def cli(argv)
         BANNER
         opts.on("--pro", "travis pro") { self.pro = true }
         opts.on("--branches", "dedup builds on branches too") { self.branches = true }
+        opts.on("--retry", "number of times to retry when Travis returns a 500. #{RETRY} times by default") { |value| self.retry_option = value }
+        opts.on("--force", "give up cancelling duplicate builds when Travis keeps returning 500") { self.force = true }
```

Comment:
When you have to restart manually 10 times a build because Travis API keeps returning 500 when running `travis_dedup`, we do need it, yes 😸